src/layouts/Base.astro

```diff
@@ -16,13 +27,14 @@ const canonicalURL = new URL(
 
 const jsonLDSchema = JSON.stringify(
 	{
-		'@context': 'http://www.schema.org',
+		'@context': 'https://www.schema.org/',
```

Comment:
🔍 Not using the `https` url scheme was causing a JSON+LD validator to throw some errors!

Reply:
Oh interesting. How do you see that error?

Reply:
My bad, I meant for there to be a hyperlink in my last comment! The validator I found is at https://json-ld.org/playground/. 

If you copy in our old JSON+LD, you’ll see an error about an invalid URL. It took some trial and error to realize that the validator seems to refuse to resolve `http:` urls. Even if a user agent still resolves them, I reckon that using https is still a good practice!